src/main/java/org/whispersystems/textsecuregcm/WhisperServerConfiguration.java

```diff
@@ -61,6 +62,10 @@
   @JsonProperty
   private AttachmentsConfiguration attachments;
 
+  @Valid
```

Comment:
Let's make this @NonNull, otherwise it'd be possible to start a server without a replication endpoint configured

Reply:
yep. will change.

Reply:
the intention was that we could roll back and disable replication to contact discovery completely by removing the section from the config, in case it's blowing up signal servers in some way. do you want that ability or not?

Reply:
I think we should be moving forward. if something's wrong we should fix it, if something's urgently wrong we can always just disable the replication piece by commenting out one line

Reply:
done in eca8e7e2724000260f04305a2b79cfba2822287a